sqlglot/optimizer/qualify_columns.py

```diff
@@ -200,8 +205,8 @@ def _expand_using(scope: Scope, resolver: Resolver) -> t.Dict[str, t.Any]:
         for column in scope.columns:
             if not column.table and column.name in column_tables:
                 tables = column_tables[column.name]
-                coalesce = [exp.column(column.name, table=table) for table in tables]
-                replacement = exp.Coalesce(this=coalesce[0], expressions=coalesce[1:])
+                coalesce_args = [exp.column(column.name, table=table) for table in tables]
+                replacement = exp.func("coalesce", *coalesce_args)
```

Comment:
Out of curiosity why was `exp.func` preferred over building `exp.Coalesce` directly?

Reply:
we always use exp.func when we can because it's more succint